spec/features/user_visits_homepage_spec.rb

```diff
@@ -9,14 +9,14 @@
 
   scenario "successfully redirects to sign in page" do
     visit root_path
-    click_link 'Sign in'
+    click_button "Sign in"
```

Comment:
I did a git pull origin then made a branch off of origin/example-home-page-layout. I think when the "Sign In" button is clicked it goes to the /sessions/new path which has "Sign In" in a H1 tag, so the test passes (which shouldn't be happening).